<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationWindow</class>
 <widget class="QWidget" name="AnimationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnimationWindow</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1061</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QGroupBox" name="CharacterSetupFrame">
      <property name="title">
       <string>Tile generator</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="Characters2Use_2">
         <item>
          <widget class="QComboBox" name="characters2choose"/>
         </item>
         <item>
          <widget class="QLabel" name="label_rowL">
           <property name="text">
            <string>Wiersz</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="row_rangeL"/>
         </item>
         <item>
          <widget class="QLabel" name="label_colL">
           <property name="text">
            <string>Kolumna</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="col_rangeL"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushCharacterL">
           <property name="text">
            <string>Insert</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacerL">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="editorFrame">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Frame Editor</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="LedMatrix" name="frameEditorL">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="cornerButtonEnabled">
             <bool>false</bool>
            </property>
            <property name="rowCount">
             <number>0</number>
            </property>
            <property name="columnCount">
             <number>0</number>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="buttonSpacerLeft">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QGroupBox" name="importExportOptions">
              <property name="title">
               <string>Import/Export</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QToolButton" name="importButton">
                 <property name="text">
                  <string>Import</string>
                 </property>
                 <property name="popupMode">
                  <enum>QToolButton::MenuButtonPopup</enum>
                 </property>
                 <property name="toolButtonStyle">
                  <enum>Qt::ToolButtonTextOnly</enum>
                 </property>
                 <property name="arrowType">
                  <enum>Qt::DownArrow</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="exportButton">
                 <property name="text">
                  <string>Export</string>
                 </property>
                 <property name="popupMode">
                  <enum>QToolButton::MenuButtonPopup</enum>
                 </property>
                 <property name="toolButtonStyle">
                  <enum>Qt::ToolButtonTextOnly</enum>
                 </property>
                 <property name="arrowType">
                  <enum>Qt::DownArrow</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="animationOptions">
              <property name="title">
               <string>Animation options</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QPushButton" name="playButton_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Play/Pause</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="insertFrameButton_2">
                 <property name="text">
                  <string>Insert frame</string>
                 </property>
                 <property name="popupMode">
                  <enum>QToolButton::InstantPopup</enum>
                 </property>
                 <property name="toolButtonStyle">
                  <enum>Qt::ToolButtonTextOnly</enum>
                 </property>
                 <property name="arrowType">
                  <enum>Qt::DownArrow</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="removeFrameButton_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Remove frame</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="storeFrameButton_2">
                 <property name="text">
                  <string>Store frame</string>
                 </property>
                 <property name="popupMode">
                  <enum>QToolButton::MenuButtonPopup</enum>
                 </property>
                 <property name="toolButtonStyle">
                  <enum>Qt::ToolButtonTextOnly</enum>
                 </property>
                 <property name="arrowType">
                  <enum>Qt::DownArrow</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="frameEditOptions">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="title">
               <string>Frame edit options</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Load frame</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="clearButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Clear frame</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="featuresButton">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Features</string>
                 </property>
                 <property name="popupMode">
                  <enum>QToolButton::InstantPopup</enum>
                 </property>
                 <property name="toolButtonStyle">
                  <enum>Qt::ToolButtonTextOnly</enum>
                 </property>
                 <property name="arrowType">
                  <enum>Qt::DownArrow</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="CustomCharacterLayout">
              <property name="title">
               <string>Draw Custom Character</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="LedMatrix" name="CustomCharacter">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <attribute name="horizontalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="verticalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LedMatrix</class>
   <extends>QTableWidget</extends>
   <header>ledmatrix.h</header>
   <slots>
    <slot>clearDisplay()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
